processor/resourceattributetransposerprocessor/README.md

```diff
@@ -1,30 +1,73 @@
 # Resource Attribute Transposer Processor
+This processor copies a resource level attribute to all individual logs or metric data points associated with the resource.
+If they key already exists, no action is taken (the attribute _**IS NOT**_ overwritten)
 
-Supported pipelines: logs, metrics
+## Minimum collector versions
+Introduced: v0.0.12
+Logs support: v1.0.0
```

Comment:
What exactly does this `Logs support` version mean? Maybe we can link releases we are referring to?